config.example.wikimedia.yaml

```diff
@@ -6,6 +6,8 @@ default_project: &default_project
   x-modules:
     - path: projects/wmf_default.yaml
       options: &default_options
+        proxy:
```

Comment:
It would be good to also have an example config just for RESTRouter, since this file now contains a lot of unused config options.

Reply:
Only the `table` configuration is unused now. Do we want to create a copy-paste of a 200 lines long file just to make the example?

Reply:
But most of the config is put in there, so I think it'll be clearer that way

Reply:
I'm not quite sure I follow regarding 'most of the config'

Reply:
Euh, I meant most of the config vars that, say, come from puppet are declared here.

Reply:
Nm...